### PR TITLE
bf: ZENKO-1183 fix scheduled crr resume

### PR DESCRIPTION
**Pull request by philipyoo**
Changes in this PR:
- Fix wrong property name used on init 😨 
- Re-schedule resume when a triggered scheduled resume
  fails. Only error that can occur is if update to
  zookeeper fails

### CODE REVIEW

**Review by JianqinWang — APPROVED** [review]

**Review by tmacro — APPROVED** [review]

File: extensions/replication/queueProcessor/QueueProcessor.js [hunk 1]
```diff
@@ -383,26 +383,35 @@ class QueueProcessor extends EventEmitter {
     }
 
     scheduleResume(date) {
+        function triggerResume() {
+            this._updateZkStateNode('scheduledResume', null, err => {
+                if (err) {
+                    this.logger.error('error occurred saving state ' +
```
Comment (bennettbuchanan) [inline extensions/replication/queueProcessor/QueueProcessor.js:396]: Maybe add a log indicating it is being rescheduled for a minute later.

File: extensions/replication/queueProcessor/QueueProcessor.js [hunk 2]
```diff
@@ -383,26 +383,35 @@ class QueueProcessor extends EventEmitter {
...
+                    'to zookeeper', {
+                        method: 'QueueProcessor.scheduleResume',
```
Comment (bennettbuchanan) [inline extensions/replication/queueProcessor/QueueProcessor.js]: Let's add the error here as well?

**Review by bennettbuchanan — APPROVED** [review]

File: extensions/replication/queueProcessor/QueueProcessor.js (hunk 1 above)
Comment (philipyoo) [reply extensions/replication/queueProcessor/QueueProcessor.js:396]: I'll update the message to:
`error occurred saving state to zookeeper for resuming a scheduled resume. Retry again in 1 minute`

File: extensions/replication/queueProcessor/QueueProcessor.js [hunk 3]
```diff
@@ -383,26 +383,37 @@ class QueueProcessor extends EventEmitter {
     }
 
     scheduleResume(date) {
+        function triggerResume() {
+            this._updateZkStateNode('scheduledResume', null, err => {
```
Comment (rahulreddy) [inline extensions/replication/queueProcessor/QueueProcessor.js:394]: This node name `scheduledResume` is the same one which was mis-spelt, correct? Why not use a constant 
`const RESUME_NODE = 'scheduledResume';`

**Review by rahulreddy — APPROVED** [review]

**Pull request merged**